pkg/collector/adapters/config_to_probe.go

```diff
@@ -51,7 +51,7 @@ const (
 func ConfigToContainerProbe(config map[interface{}]interface{}) (*corev1.Probe, error) {
 	serviceProperty, ok := config["service"]
 	if !ok {
-		return nil, errNoService
+		return nil, ErrNoService
```

Comment:
Why is this changed from `errNoService` to `ErrNoService`. I see that all ref are in the `adapters` package.

Reply:
I'll check it.